build_bot/common/cmd_utils.py

```diff
@@ -0,0 +1,63 @@
+import os
+import subprocess
+import tempfile
+
+
+def shell(cmd, env=None):
+    print("calling command: " + str(cmd))
```

Comment:
To use "print" is not very good practice in our task. We will need to move our print streams to web -> may be it would be better to create your own function printout which would be do the same thing. But if we need to print all our output to the file we can change printout only in one place but not in several places of code.